cli/hzn.go

```diff
@@ -564,6 +565,18 @@ Environment Variables:
 	mmsObjectDownloadId := mmsObjectDownloadCmd.Flag("id", msgPrinter.Sprintf("The id of the object to download data. This flag must be used with -t.")).Short('i').Required().String()
 	mmsObjectDownloadFile := mmsObjectDownloadCmd.Flag("file", msgPrinter.Sprintf("The file that the data of downloaded object is written to. This flag must be used with -f. If omit, will use default file name in format of objectType_objectID and save in current directory")).Short('f').String()
 
+	voucherCmd := app.Command("voucher", msgPrinter.Sprintf("List and manage Horizon SDO ownership vouchers."))
+
+	voucherInspectCmd := voucherCmd.Command("inspect", msgPrinter.Sprintf("Display properties of the SDO ownership voucher."))
+	voucherInspectFile := voucherInspectCmd.Arg("voucher-file", msgPrinter.Sprintf("The SDO ownership voucher file.")).Required().File()  // returns the file descriptor
+
+	voucherImportCmd := voucherCmd.Command("import", msgPrinter.Sprintf("Imports the SDO ownership voucher so that the corresponding device can be booted, configured, and registered. HZN_SDO_SVC_URL must be set in the environment, /etc/default/horizon, or one of the hzn.json files."))
+	voucherImportFile := voucherImportCmd.Arg("voucher-file", msgPrinter.Sprintf("The SDO ownership voucher file. Must be file type extension: json, tar, tar.gz, tgz, or zip. If it is any of the tar/zip formats, all json files within it will be imported (other files/dirs will be silently ignored).")).Required().File()  // returns the file descriptor
+	voucherOrg := voucherImportCmd.Flag("org", msgPrinter.Sprintf("The Horizon organization ID. If not specified, HZN_ORG_ID will be used as a default.")).Short('o').String()
+	voucherUserPw := voucherImportCmd.Flag("user-pw", msgPrinter.Sprintf("Horizon user credentials to import a voucher. If not specified, HZN_EXCHANGE_USER_AUTH will be used as a default. If you don't prepend it with the user's org, it will automatically be prepended with the -o value.")).Short('u').PlaceHolder("USER:PW").String()
+	voucherImportExample := voucherImportCmd.Flag("example", msgPrinter.Sprintf("The example edge service that should be run on this device, for example: helloworld. It is mutually exclusive with --policy.")).Short('e').String()
```

Comment:
1) The description of example confused me until I found the code. Can we say that it sets a policy property on the node that is used to enable deployment of an example service?
2) In the description you use the them "this device", but I thought the command was being used by an admin who was not physically on the edge device, so then what does "this device" refer to? I think you mean the device whose voucher you are importing?

Reply:
yes, corrected